handlers/hjt/hjt.go

```diff
@@ -0,0 +1,81 @@
+package hjt
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/lordralex/absol/database"
+	"github.com/lordralex/absol/logger"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, c *discordgo.Channel, cmd string, args []string) {
+	if len(args) == 0 {
+		return
+	}
+
+	pasteLink := args[0]
+
+	content, err := readFromUrlAsLowercase(pasteLink)
+	if err != nil {
+		_, err = ds.ChannelMessageSend(c.ID, "Invalid URL")
+		logger.Debug().Printf("Invalid url: %s\n%s", pasteLink, err)
+		return
+	}
+
+	db, err := database.Get()
+	if err != nil {
+		_, err = ds.ChannelMessageSend(c.ID, "Failed to connect to database")
+		logger.Err().Printf("Failed to connect to database\n%s", err)
+		return
+	}
+
+	var data []hjt
+	err = db.Find(&data).Error
```

Comment:
Why aren't we just using the DB to handle this concept? You can do a SELECT with a LIKE. This is going to be slow and eat way more memory than it should.

Reply:
I thought LIKE only checked if what you are passing its contained in database, this has to be the opposite way(from our big chunk of text, we want to see what in database is contained there)

I suck at SQL so if you have any suggestions please tell me :D

Reply:
You can do the reverse:

'myteststring' LIKE '*' + column + '*'

Reply:
PErfect, will try. Can you make sure the query limit in mysql is not too low?

Reply:
Done